addons/mail/static/src/discuss/core/common/thread_model_patch.js

```diff
@@ -298,7 +316,9 @@ const threadPatch = {
         }
         const previousState = this.fetchMembersState;
         this.fetchMembersState = "pending";
-        const known_member_ids = this.channel_member_ids.map((channelMember) => channelMember.id);
+        const known_member_ids = this.channel?.channel_member_ids.map(
```

Comment:
either ensure empty array, or remove guard (I think there should be necessarily a channel in this flow)